EIPS/eip-1.md

```diff
@@ -42,19 +42,23 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
 
-If the EIP collaborators approve, the EIP editor will assign the EIP a number (generally the issue or PR number related to the EIP), label it as Standards Track, Informational, or Meta, give it status “Draft”, and add it to the git repository. The EIP editor will not unreasonably deny an EIP. Reasons for denying EIP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the Ethereum philosophy.
+Drafts must be formatted as described in this EIP; a template is available in [eip-X.md](https://github.com/ethereum/EIPs/blob/master/eip-X.md). All required headers must be present and filled in correctly. For a draft, the `eip` field must match the filename, without the 'eip-' prefix or '.md' suffix (eg, for `eip-draft-vitalik-foo.md`, the `eip` field should read `draft-vitalik-foo`). Drafts must contain a `discussions-to` header, specifying the URL to a venue where collaborators can discuss the draft. This may be a Github issue in the EIPs repository, or any other venue as the author sees fit.
 
-Standards Track EIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification]. The EIP should be reviewed and accepted before an implementation is begun, unless an implementation will aid people in studying the EIP. Standards Track EIPs must be implemented in at least three viable Ethereum clients before it can be considered Final.
+An automated process will check all pull requests; if the request passes the format checks and contains only additions or edits to drafts owned by the submitter, it will be automatically merged. No editorial control is exercised over the content of an EIP draft.
 
-For an EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+Standards track EIPs of type 'Core' are updated from 'Draft' to 'Accepted' status when a core devs meeting has accepted the draft and the participants have announced their intention to implement the EIP for inclusion in a future hard fork. At that point, an EIP number is assigned by the editors and the EIP is moved to the EIPS directory and renamed accordingly. Once a Core EIP has been implemented in at least three clients, and all pass a common set of test suites, the status of the EIP is updated to 'Final'. An EIP can also be assigned status “Deferred”. The EIP author or editor can assign the EIP this status when no progress is being made on the EIP.
```

Comment:
This seem very late in the process to assign a number.  RFCs get a number much earlier. (BIPs and PIPs too, I think).  Without a number it's hard to unambiguously refer to a Draft under discussion.

Reply:
I'm actually modelling this after the RFC process; RFCs only get a number after the working group is done with them.

Reply:
We don't have working groups as an explicit part of our process, and the IETF doesn't have coredevs.  The closest we might have to working groups would be groups in the Fellowship, and their work is done when an EIP becomes a Draft.  I think this is about the stage at which an RFC gets a number, not the stage at which the whole internet is sure to go with it.

Reply:
The other stage where a number might be assigned is my proposed Ready status.  The editors, the authors, and any ad hoc working groups (possibly including Fellowship groups) a quite done with it, and as far as we are concerned it's a thing.

Reply:
I don't think that's a reasonable analogy. In the IETF, an RFC gets a number when editing is basically finished. When a draft is submitted, it's barely begun. I don't think it makes sense, or is necessary, to assign EIP numbers to every draft.

Reply:
The Accepted and Final status are just an after-the-fact bookkeeping service we provide to the current core devs.  I anticipate a world of multiple blockchains accepting different sets of EIPs,  all attempting to follow the Ethereum standard rather than just be contentious forks.

Reply:
If others want to have their own statuses for their EIPs, they're welcome to maintain that. I don't think that argues for assigning numbers to EIPs while they're still being actively modified.

Reply:
Are RFCs not actively modified once numbered?  (That's not a rhetorical question.)

Reply:
Oh, you answered above, but I failed to reload the page.

Reply:
They're not: https://www.rfc-editor.org/wp-content/uploads/rfc-editor-process.gif

I would hope, too, that submitting drafts here is treated as an early step, too - not the end result of a conversation on Ethereum Magicians, but a way to start one.

Reply:
I guess I don't see the point of calling something a Request for Comments when in fact the document is done and not open to further comments.  But I don't know their process that well.  Anyway, this is again why I propose a Ready status as the right place for us to say our work is done, so here is a proposal, EIP-###.  The path from there to being accepted can be long and contentious, and it really will help to have a document number during that process.

Reply:
You'd have to ask the original IETFers about that.

I still don't see the need for a separate 'Ready' state; if you want your Core EIP moved to 'Accepted', ask for it to be added to the all-core-devs agenda, and if you want your non-core EIP moved, ask an editor. Neither one requires a state just for that.

Reply:
That is, without numbering EIPs until after they are accepted the discussion of EIPs can go like...
"The core devs accepted Nick's proposal."
"Which one?"
"Something like nicks_minimal_troll_vaporization_mechanism". 
"You mean the one he presented at NevCon7?"
"No, the one after that, that vaporized fewer cases."
"What's its EIP number?"
"I don't know.  It hasn't been assigned yet."

Reply:
Is it that difficult to talk about "draft-nickjohnson-lets-vaporize-trolls"?

Reply:
I understand this conversation happened over 1 year ago.


In my experience in other context, having an ID as early as possible actually helps a lot for disambiguation. What's the benefit of not assigning number, other than feeling not all draft needs number? Is it to conserve number space? @Arachnid